DEVELOPERS_GUIDE.md

```diff
@@ -1,48 +1,48 @@
-# Developer's Guide
+# KaiZen OpenAPI Editor - Developer's Guide
 
-# Building SwagEdit from sources
-Development of SwagEdit should be done with Eclipse. This project uses Maven/Tycho so you should have the Eclipse maven plugin `m2e` already installed.
+# Building from Sources
+Development of KaiZen Editor should be done with Eclipse. This project uses Maven/Tycho so you should have the Eclipse maven plugin `m2e` already installed.
 
 ## 1. Get sources
-To start developing SwagEdit, clone the repository with the following command:
+To start developing, clone the repository with the following command:
 
 ```
-git clone git@github.com:ModelSolv/SwagEdit.git
+git clone git@github.com:RepreZen/KaiZen-OpenAPI-Editor.git
 ``` 
 
 ## 2. Import to Eclipse IDE
-Open Eclipse and select `File > Import... > Maven > Existing Maven Project` and select the folder SwagEdit.
-This will put the project SwagEdit into your current workspace. 
+Open Eclipse and select `File > Import... > Maven > Existing Maven Project`. Then select the folder KaiZen-OpenAPI-Editor.
+This will add the project into your current workspace. 
```

Comment:
> add the project

It's several project*s*, plural.
We have three projects now + a test plugin + target platform. Here is a screenshot showing which projects should be selected:
<img width="663" alt="screen shot 2017-05-09 at 11 28 15 am" src="https://cloud.githubusercontent.com/assets/644582/25858815/b3248a80-34aa-11e7-826b-e6b5547a01ed.png">